### PR TITLE
Release 1.1.facilities fixes

### PR DESCRIPTION
**Pull request by delarosatrevin**
### Summary of the changes in this PR:
* Relion:
  * Changed how the Magnification is handled in Relion convert.py 
  * Some fixes in the bibtex
  * Changes in how input Alignment is treated (plus added priors)
  * Bug fixed in protocol base using 'realignMovieFrames' that is not defined in classify2D
  * Bug fixed in extract-particles when extracting from a subset
* Ctffind4 protocol now allow to set Yes/No for slow/fast search 
* In scipion script the 'v' of the version name is used in lowercase for more consistent naming
* Some values persisted in screen-particles protocol to be used in summary and test validation.
* Add radius size validation in extract movie particles protocol

### CODE REVIEW

File: pyworkflow/em/packages/relion/convert.py [hunk 1]
```diff
@@ -359,6 +359,9 @@ def alignmentToRow(alignment, alignmentRow, alignType):
     if is2D:
         angle = angles[0] + angles[2]
         alignmentRow.setValue(md.RLN_ORIENT_PSI, -angle)
+        # Also set the prior
+        alignmentRow.setValue(md.RLN_ORIENT_ROT_PRIOR, -angle)
```
Comment (azazellochg) [inline pyworkflow/em/packages/relion/convert.py]: should this be md.RLN_ORIENT_PSI_PRIOR ?

**Comment by azazellochg** [comment]
Would it be reasonable address #1402 here? Or you want to postpone it?

**Comment by delarosatrevin** [comment]
Yes, I think it should be addressed here. I thought that this was taken into account by @josuegbl in the changes related to Relion. @josuegbl Can you double-check?

**Comment by pconesa** [comment]
@delarosatrevin and @josuegbl ....I've pushed my fixes for the screen particle. I didn't like the summaryVars thing since then is "very dirty" to test that those values are right, even if only used in the summary. If scipion is used as an API....this way, these values could be easily retrieved. Hope you don't mind to leave it as it is.

File: pyworkflow/em/packages/relion/convert.py (hunk 1 above)
Comment (josuegbl) [reply pyworkflow/em/packages/relion/convert.py]: you are right @azazellochg, it was an error introducing the label :)

**Comment by delarosatrevin** [comment]
@josuegbl Can you take a look at this PR and merge before it continues to grow? Some other minor changes has been introduced since @pconesa approved it. ;)

**Pull request merged**